File: easy/Bit-Positions/Bit Positions.cs
using System;
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            ShowBitPos(line);
        }
    }
    
    static void ShowBitPos(string line){
        string[] nums = line.Split(',');
        int num = Convert.ToInt32(nums[0]);
        int pos1 = Convert.ToInt32(nums[1]);
        int pos2 = Convert.ToInt32(nums[2]);
        string binary = Convert.ToString(num, 2);
        int leng = binary.Length;
        Console.WriteLine(binary[leng-pos1].Equals(binary[leng-pos2]).ToString().ToLower());
    }
    
}

File: easy/Bit-Positions/Bit-Positions.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            ShowBitPos(line);
        }
    }
    
    static void ShowBitPos(String line){
        String[] nums = line.split(",");
        int num = Integer.valueOf(nums[0]);
        int pos1 = Integer.valueOf(nums[1]);
        int pos2 = Integer.valueOf(nums[2]);
        String binary = Integer.toBinaryString(num);
        int leng = binary.length();
        System.out.println(binary.charAt(leng-pos1) == binary.charAt(leng-pos2));
    }
}
